### PR TITLE
Add LXC: Iventoy

### PR DESCRIPTION
**Pull request by BramSuurdje**
## Description

Added a Iventoy script.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by tteck** [comment]
Have you considered

```
RELEASE=$(curl -s https://api.github.com/repos/ventoy/pxe/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
msg_info "Installing iVentoy v${RELEASE}"
mkdir -p /opt/iventory
wget -qO- https://github.com/ventoy/PXE/releases/download/v${RELEASE}/iventoy-${RELEASE}-linux-free.tar.gz | tar -xzf - -C /opt/iventory --strip-components=1
echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
msg_ok "Installed iVentoy v${RELEASE}"
```

**Comment by manuelkue** [comment]
Would it be possible (in the future) to include either more information in the description or simply a link to the tool's homepage when opening an MR with some new application?

Perhaps that would be useful also for others that like to be up to date with the most recent additions but then have to search for background information about potentially new cool features :-)

**Comment by BramSuurdje** [comment]
when new scripts get released they will appear on the home page of [the website](https://helper-scripts.com/scripts). from there you can click on the script and view a more detailed description about what the program does. and links to documentation and the website.

File: install/iventoy-install.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+msg_info "Installing iVentoy v${RELEASE}"
+mkdir -p /opt/iventoy/
+wget -q https://github.com/ventoy/PXE/releases/download/v${RELEASE}/iventoy-${RELEASE}-linux-free.tar.gz
+tar -C /tmp -xzf iventoy*.tar.gz
+mv /tmp/iventoy*/* /opt/iventoy/
+rm -rf iventoy*.tar.gz
+msg_ok "Installed iVentoy"
+
+msg_info "Creating Service"
+service_path="/etc/systemd/system/iventoy.service"
```
Comment (tteck) [inline install/iventoy-install.sh]: What's this for?

File: ct/iventoy.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  MTU=""
+  SD=""
+  NS=""
+  MAC=""
+  VLAN=""
+  SSH="no"
+  VERB="no"
+  echo_default
+}
+
```
Comment (tteck) [inline ct/iventoy.sh:54]: function update_script() {
  if [[ ! -d /opt/iventoy ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
  msg_error "There is currently no update path available."
  exit  
}

File: install/iventoy-install.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Environment       =IVENTOY_AUTO_RUN=1
+Environment       =LIBRARY_PATH=/opt/iventoy/lib/lin64
+Environment       =LD_LIBRARY_PATH=/opt/iventoy/lib/lin64
+ExecStart         =sh ./iventoy.sh -R start
+WorkingDirectory  =/opt/iventoy
+Restart           =on-failure
+[Install]
+WantedBy=multi-user.target
+EOF
+systemctl enable --now iventoy.service
```
Comment (tteck) [inline install/iventoy-install.sh]: systemctl enable -q --now iventoy.service

File: install/iventoy-install.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+msg_ok "Created Service"
+
+motd_ssh
+customize
+
+msg_info "Cleaning up"
+$STD apt-get autoremove
```
Comment (tteck) [inline install/iventoy-install.sh]: $STD apt-get -y autoremove
$STD apt-get -y autoclean

**Pull request merged**